update manual and docstrings to PrettyTables.jl (#2522)
JuliaData/DataFrames.jl: docs/src/man/getting_started.md

julia> df = DataFrame(x=x)
3×1 DataFrame
 Row │ x
     │ Int64
─────┼───────
   1 │     3
   2 │     1
   3 │     2

julia> sort!(df)
3×1 DataFrame
 Row │ x
     │ Int64
─────┼───────
   1 │     1
   2 │     2
   3 │     3

julia> x
3-element Array{Int64,1}:
 3
 1
 2

julia> df.x[1] = 100
100

julia> df
3×1 DataFrame
 Row │ x
     │ Int64
─────┼───────
   1 │   100
   2 │     2
   3 │     3

> julia> x

Reviewer: While we're at it, probably better not print `x` here as it's not mutated, which is kind of the opposite of what the text above says. Or we should explain that `DataFrame(x=x)` makes a copy.

Author: I think this was the point of the original example. I have added a comment below.

 1
 2
```
Note, that in the above example the original `x` vector is not mutated in the

> process as `DataFrame(x=x)` constructor stores its copy in a data frame.

Reviewer: ```suggestion
process as the `DataFrame(x=x)` constructor makes a copy by default.
```
